client/tsconfig.base.json

```diff
@@ -21,7 +21,8 @@
       "@client/hooks": ["modules/_hooks/src/index.ts"],
       "@client/onboarding": ["modules/onboarding/src/index.ts"],
       "@client/test-fixtures": ["modules/_test-fixtures/src/index.ts"],
-      "@client/workspace": ["modules/workspace/src/index.ts"]
+      "@client/workspace": ["modules/workspace/src/index.ts"],
+      "@client/workspace/*": ["modules/workspace/src/*"]
```

Comment:
Let's remove the wildcard import here. I think it might cause some issues with the Nx tooling, and it's only being used for a single CSS module file currently.